### ISSUE TITLE
Updated documentation or Wiki

### ISSUE THREAD

**Issue by julianjupiter** [eclipse-ee4j/glassfish-grizzly#2100]
The website, https://javaee.github.io/grizzly/, remains temporarily paused until now. I'm struggling looking for the updated documentation of Grizzly (as well as HK2). The link of status provided returns 404: https://www.eclipse.org/ee4j/status.php.
![grizzly-01](https://user-images.githubusercontent.com/15760709/85065837-73b12000-b1e0-11ea-98f0-76062a09194a.JPG)
![grizzly-02](https://user-images.githubusercontent.com/15760709/85065840-7449b680-b1e0-11ea-8333-87f9c77fe41a.JPG)

**Comment by tomqwpl** [comment]
Ditto. Unclear what the status of this project is

**Comment by julianjupiter** [comment]
Not sure, but I've noticed many of Eclipse projects have bad, outdated or no documentations. I'm afraid that the projects transferred alongside Java EE such as Grizzly, HK2, Jersey, etc. would have the same fate. I could be wrong though.

**Comment by smillidge** [comment]
The project is active although much of the documentation was not transferred over to Eclipse.
The project page is https://projects.eclipse.org/projects/ee4j.grizzly

**Comment by tomqwpl** [comment]
The situation is very confusing though. The documentation that's easy to find makes it look like the project has been abandoned. The pages have the banner at the bottom saying activity has stopped while it's transferred, but none of the links works. There's no obvious documentation here on github, or on the eclipse project page. 
Yet there is still commit activity. There has been at least a 2.4.4 release, and there's an upcoming 3.0 release, but I can find no information about what might be in those.

**Comment by smillidge** [comment]
I'm not a project committer and I agree. However, the project has now moved to Eclipse without the documentation. Most of the maintainers likely use Grizzly as a component of bigger projects. I imagine everybody is focussed on Jakarta EE 9 work and 3.0 will be the Jakarta EE 9 release.

**Comment by julianjupiter** [comment]
I think it's becoming a huge debt as more releases are going to be focused on.

**Comment by edbratt** [comment]
I've merged my own PR (above). The doc source and trace details to get going are in that orphan [gh-pages branch](https://github.com/eclipse-ee4j/grizzly/tree/gh-pages) readme.md. I've submitted [an issue](https://bugs.eclipse.org/bugs/show_bug.cgi?id=572123) for the Eclipse Web Admins to enable the site. Once that's turned on, I plan to close this issue. Starter problems for community to tackle are written in the branch readme. Of course, they could also be filed as issues here.
